r/README.Rmd

```diff
@@ -24,8 +25,10 @@ First install a release build of the C++ bindings to arrow.
 git clone https://github.com/apache/arrow.git
 cd arrow/cpp && mkdir release && cd release
 
-# It is important to statically link to boost libraries
-cmake .. -DARROW_PARQUET=ON -DCMAKE_BUILD_TYPE=Release -DARROW_BOOST_USE_SHARED:BOOL=Off
+# It is important to statically link to boost libraries.
+# For this, you may need to use a vendored version of the
+# Boost libraries (e.g. on Ubuntu).
+cmake .. -DARROW_PARQUET=ON -DCMAKE_BUILD_TYPE=Release -DARROW_BOOST_USE_SHARED:BOOL=Off -DARROW_BOOST_VENDORED=ON
```

Comment:
Vendoring Boost makes it risky to symbol collisions, we should try and namespace our own boost: https://issues.apache.org/jira/browse/ARROW-4286